stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/home/HomeFragmentViewModel.kt

```diff
@@ -42,6 +43,17 @@ class HomeFragmentViewModel : ViewModel() {
             user = ChatClient.instance().getCurrentUser() ?: unauthorizedUser,
```

Comment:
I believe this could be replaced now with 
`ChatClient.instance().clientState.user.value ?: unauthorizedUser.`